Formula/etcher-cli.rb

```diff
@@ -0,0 +1,32 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher/archive/v1.4.4.tar.gz"
+  sha256 "02082bc1caac746e1cdcd95c2892c9b41ff8d45a672b52f8467548cad4850f5d"
+
+  depends_on "node@6" => :build # must be abi compatiable to the node version used with pkg
+  depends_on "python" => :build
+
+  def install
+    ENV["NODE_ENV"] = "production"
+    system "npm", "install", *Language::Node.local_npm_install_args
+    inreplace buildpath/"node_modules/lzma-native/index.js",
+              "var native = require(binding_path);",
+              "var native = require(path.join(__dirname, 'binding', 'lzma_native.node'));"
+
+    system "npm", "install", *Language::Node.local_npm_install_args, "pkg@4.3.0", "--prefix=#{buildpath}/pkg"
+
+    system buildpath/"pkg/node_modules/.bin/pkg", "--output", libexec/"etcher", "--build",
```

Comment:
BTW: If you want faster builds, temporary comment out the `"--build",` to use the prebuild `pkg` node binaries for testing. But remember to comment it back in at the end, because we want to compile everything from source for the final bottle.